addons/debugger/style.css

```diff
@@ -35,6 +35,9 @@
   background-color: white;
   width: 565px;
   height: 25rem;
+  resize: both;
+  min-width: 565px;
+  min-height: 25rem;
```

Comment:
What if you want to make it smaller? I think that's the main want currently, it's quite big

Reply:
I think I should make this thing a setting and those values customizable

Reply:
It doesn't make sense to customize the minimum dimensions.
The minimum dimensions should be the lowest ones where the buttons still fit and our CSS doesn't break.

Reply:
So **I** have to manually put that values

Reply:
> So **I** have to manually put that values

Correct.

Reply:
@WorldLanguages Is it a good size (height: 335px and width: 483px)?

![image](https://user-images.githubusercontent.com/78246772/205438191-fa6c0e6b-5b59-4e64-b068-42e1ba130454.png)

Reply:
@Platformer444 Personally I'd round the width to 450px and height to 300px

Reply:
> @Platformer444 Personally I'd round the width to 450px and height to 300px

Does that work? You're rounding down. (Just making sure before something theoretically breaks.)

Also, why does rounding matter? The minimum dimensions should be the minimum dimensions possible, not some number evenly divisible by 50 just for the sake of being evenly divisible by 50. The difference between what's allowed and what's possible might matter to some people.

Reply:
> You're rounding down.

Sorry, you're right. The min width can't be 450px

Reply:
@WorldLanguages Is this good now(height: 300px and width: 483px)?
![image](https://user-images.githubusercontent.com/78246772/205478550-2fd36579-a4c4-4fa2-9a87-2f1989819f06.png)

Reply:
Does the minimum width work in languages with longer button labels?

Reply:
> Does the minimum width work in languages with longer button labels?

Oh didn't see that

Reply:
The debugger window doesn't seem to be translated.

Reply:
@Samq64 It is - you need to reload the page after changing the language.
![image](https://user-images.githubusercontent.com/51849865/206653797-9d1e8f83-9ab4-4002-9fa8-273535468e06.png)